src/Donors/resources/admin/components/DonorDetailsPage/Tabs/Overview.tsx

```diff
@@ -1,7 +1,222 @@
+import React from 'react';
+import {__} from '@wordpress/i18n';
+import {dateI18n} from '@wordpress/date';
+import classnames from 'classnames';
+import StatWidget from '@givewp/src/Admin/components/StatWidget';
+import Header from '@givewp/src/Admin/components/Header';
+import PrivateNote from '@givewp/src/Admin/components/PrivateNote';
+import TimeSeriesChart from '@givewp/src/Admin/components/Charts/TimeSeriesChart';
+import {useDonorStatistics} from '@givewp/donors/hooks/useDonorStatistics';
+import {amountFormatter, formatTimestamp, getRelativeTimeString} from '@givewp/src/Admin/utils';
+import {getDonorOptionsWindowData, useDonorEntityRecord} from '@givewp/donors/utils';
+import {useDonorDonations} from '@givewp/donors/hooks/useDonorDonations';
+
+import styles from '@givewp/donors/admin/components/DonorDetailsPage/DonorDetailsPage.module.scss';
+
+/**
+ * @unreleased
+ */
+type Transaction = {
+    campaign: string;
+    status: 'Completed' | 'Pending' | 'Failed' | 'Refunded';
+    timestamp: string;
+    amount: string;
+};
+
+/**
+ * @unreleased
+ */
+const statusMap: Record<string, 'Completed' | 'Failed' | 'Pending' | 'Refunded'> = {
+    publish: 'Completed',
+    completed: 'Completed',
+    processing: 'Pending',
+    pending: 'Pending',
+    failed: 'Failed',
+    cancelled: 'Failed',
+    abandoned: 'Failed',
+    preapproval: 'Pending',
+    revoked: 'Failed',
+    refunded: 'Refunded',
+};
+
+/**
+ * @unreleased
+ */
 export default function DonorDetailsPageOverviewTab() {
+    const urlParams = new URLSearchParams(window.location.search);
+    const donorId = parseInt(urlParams.get('id') ?? '0');
+    const {currency, mode} = getDonorOptionsWindowData();
+    const {statistics: stats, isResolving: statsLoading, hasResolved: statsResolved,} = useDonorStatistics(donorId, mode);
+    const {donations} = useDonorDonations({donorId, mode});
+    const {record: donor} = useDonorEntityRecord(donorId);
+    const donationChartEndpoint = `givewp/v3/donations?mode=${mode}&donorId=${donorId}`;
+    const donationsListUrl = `admin.php?page=give-payment-history&donor=${donorId}`;
+
+    const transactions: Transaction[] = !donations
+        ? []
+        : donations.map((donation) => ({
+              campaign: donation.formTitle,
+              status: statusMap[donation.status] || 'Pending',
+              timestamp: donation.createdAt.date,
+              amount: amountFormatter(donation.amount.currency).format(parseFloat(donation.amount.value)),
+          }));
+
+    const summaryItems = !stats
+        ? []
+        : [
+              {
+                  label: __('Donor Since', 'give'),
+                  value: dateI18n('M j, Y', donor?.createdAt.date, undefined),
+              },
+              {
+                  label: __('Last Contributed', 'give'),
+                  value: stats.donations.first
+                      ? getRelativeTimeString(new Date(stats.donations.first.date))
+                      : __('Never', 'give'),
```

Comment:
This should be using the `stats.donations.last` value.  I suggest keeping this consistent with the shape and implementation of the `status.donations.first` property